app/resources/css/components/vote-result-chart.css

```diff
@@ -35,3 +35,13 @@
 .vote-result-chart__bar--abstention {
     background-color: var(--blue);
 }
+
+@media screen and (max-width: 1024px) {
+    .vote-result-chart__bar[ratio='medium-ratio'] > svg {
+        display: none;
+    }
+
+    .vote-result-chart__bar[ratio='small-ratio'] {
+        color: transparent;
```

Comment:
Damit nimmt der Text trotzdem noch Platz weg, oder?

Wie wäre es, ihn einfach in einem `span` zu wrappen und dann auch `display: none` zu verwenden?